### PR TITLE
[Manual Taxes #1] Analytics events

### LINKED ISSUE #9668: Analytics events

### PR DESCRIPTION
**Pull request by samiuelson**
Analytics events for Manual Taxes:
1. Tap on question mark to show the educational dialog -> "order_taxes_help_button_tapped" 
2. Tap on Edit Tax Rates in Admin button on educational dialog -> "tax_educational_dialog_edit_in_admin_button_tapped"

Closes: #9668

### Testing instructions

Verify the above events are tracked, when:
1. Clicking on "?" icon in order creation 
2. Clicking "Edit Tax Rates in Admin" in the Tax information dialog opened in step 1.

### Images/gif

Screenshot 2023-08-25 at 22 16 16

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt [hunk 1]
```diff
@@ -1102,6 +1103,9 @@ class OrderCreateEditViewModel @Inject constructor(
                 )
             )
         )
+        tracker.track(AnalyticsEvent.ORDER_TAXES_HELP_BUTTON_TAPPED, mapOf(
+            KEY_SOURCE to "order_creation"
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt]: `"order_creation"` is tracked even when that' order editing flow. Is this expected?

In the same VM we track 

```
mapOf(KEY_FLOW to flow)
```

Won't it make sense to do the same here? Not 100% understand what is the goal here to track "source" with the same parameter every time

**Review by kidinov — COMMENTED** [review]
Tracking works! I left a comment to reconsider what we track though

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt (hunk 1 above)
Comment (samiuelson) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt]: `KEY_FLOW` is used with "creation" or "editing" values. In this case, the values will be "order_creation" and "order_creation_customer_details" (the info "?" icon will be shown in customer details as well). We discussed it here: p1692975304691539-slack-C025A8VV728

Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditViewModel.kt]: oh, I see! Thanks.

Don't you think that this is confusing that you currently track `source=order_creation` for both order creation and order editing cases? Don't we want to know if this called from order creating or order editing screens?

Also, np, but looking into the code base we usually use constants from `AnalyticsTracker`, e.g. `AnalyticsTracker.KEY_SOURCE to AnalyticsTracker.VALUE_AZTEC_EDITOR`

**Comment by samiuelson** [comment]
> Tracking works! I left a comment to reconsider what we track though

Thanks for the review, @kidinov. 

We agreed, to remove tracking of the KEY_SOURCE prop completely because we decided to remove the customer details screen from the project's scope. More context: p1693216850260329/1692975304.691539-slack-C025A8VV728

**Review by kidinov — APPROVED** [review]
LGTM!

**Pull request merged**